src/conv/invokers/mlir_impl_gemm.cpp

```diff
@@ -69,6 +69,32 @@ auto PermuteDimStride(const std::vector<size_t>& dims, const std::vector<size_t>
     return std::make_tuple(sorted_dims, sorted_strides);
 };
 
+auto InsertGToDimStride(const std::string& layout,
```

Comment:
[Recommendation] InsertGToDimStride -> InsertGToDimsStrides or InsertNumGroupsToDimsStrides

Reply:
This function is only dealing with one pair of dim/stride, therefore I used the single form.

Reply:
Hmm... I thought that stuff like {10,20,40,80} is strides but {20} is stride. In other words, tensor is defined by strides, not by stride?

Reply:
True, I updated all naming to be `strides` or `dims`.